src/person/controllers/update_controller.py

```diff
@@ -23,6 +27,12 @@ def process(self):
         update(self._person, "grade", grade)
         update(self._person, "phone_number", phone_number)
         update(self._person, "pronouns", pronouns)
+
+        if profile_pic_base64 is not None:
+            upload_profile_pic(self._user.id, profile_pic_base64)
+            self._user = User.objects.get(id=self._user.id)
+            return success_response(self._serializer(self._user).data)
```

Comment:
if the client is updating the profile picture, do you know that they aren't going to be updating anything else? if yeah then this is valid but if they can update multiple things at once then this will return before other things are updated - just checking

Reply:
kate should verify that `profile_pic_url` isn't already set at line 31

Reply:
oh